packages/common/src/users/_internal/UserUiSchemaSettings.ts

```diff
@@ -16,6 +18,47 @@ export const buildUiSchema = async (
   options: any,
   context: IArcGISContext
 ): Promise<IUiSchema> => {
+  let associatedOrgName;
+  let noticeMessage = `{{${i18nScope}.notice.message:translate}}`;
+  // default Notice action
```

Comment:
maybe make this more explicit: "default notice action - "Go to organization settings""